templates/stamp/utilities.sh

```diff
@@ -1698,3 +1700,75 @@ copy_bits()
     ssh -o "${ssh_options}" "${bitscopy_target_user}@${bitscopy_target_server}" "sudo chmod 600 ~/install.sh && sudo chmod 600 ~/utilities.sh"
     exit_on_error "Unable to update permissions on the installer files copied to '${bitscopy_target_server}'!" "${error_code}" "${copyerror_mail_subject}" "${copyerror_mail_receiver}"
 }
+
+#############################################################################
+# Memcache
+#############################################################################
+
+install-memcached()
+{
+    if type memcached >/dev/null 2>&1; then
+        log "Memcached is already installed"
+    else
+        log "Updating Repository"
+        apt-get update -y -qq
+
+        log "Installing Memcached"
+        apt-get install -y memcached
+        exit_on_error "Failed to install the Memcached on ${HOSTNAME} !" $ERROR_MEMCACHED_INSTALLER_FAILED
+
+        log "Installed Memcached successfully."
+    fi
+}
+
+install-pip()
+{
+    if type pip >/dev/null 2>&1; then
+        log "PIP is already installed"
+    else
+        log "Updating Repository"
+        apt-get update -y -qq
+
+        log "Installing Memcached"
+        apt-get install -y python-pip python-dev build-essential
```

Comment:
consider using the new install wrapper after you merge in the latest changes from oxa/dev.fic

Reply:
Sure, i'll consider it...